README.md

```diff
@@ -13,6 +13,12 @@ This is a template [Flask](https://flask.palletsprojects.com) app using the [GOV
 
 The app is provided intentionally bare, with just the essential parts that all services need, such as error pages, accessibility statement, cookie banner, cookie page and privacy notice. It uses a number of other packages to provide the [features](#features) described below with sensible and best-practice defaults. Please read the [next steps](#next-steps) section for guidance on how to start building out your app on top of this template.
 
+# Versioning 
+- Python: 3.12
+- Node.js: 22.12.0
+- npm: 10.9.0
+- Pip: 24.3.1
```

Comment:
pip should either be in full lowercase or full upper case, just like npm.

Reply:
pip should be lowercase